src/sync.d

```diff
@@ -355,6 +355,10 @@ final class SyncEngine
 	{
 		log.vlog(item.id, " ", item.name);
 		string path = itemdb.computePath(item.id);
+		if (!isValidName(path)) {
+			log.log("The item: ", path, " is invalid name. It's not uploaded.");
```

Comment:
Could you change the log string to `log.log("Invalid name, skipping: ", path);` for consistency with other log messages?